config/environments/development.rb

```diff
@@ -6,7 +6,7 @@
   # In the development environment your application's code is reloaded any time
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.enable_reloading = true
```

Comment:
Rails changed the name of this option and reversed is, so enable_reloading=!cache_classes